glossary.yml

```diff
@@ -2887,6 +2887,12 @@
     term: ትዕዛዝ-መስመር አገናኝ
     def: >
       ኢንተርፌት በ[ዛጎል](#shell) ውስጥ ለመሮጥ ትዕዛዞች እና ውጤቶች ለማግኘት በጽሁፍ ላይ ብቻ የሚደገፍ  ።
+  uk:
+    term: інтерфейс командного рядка
+    def: >
+      Інтерфейс користувача, який повністю покладається на текст
+      для введення команд та виведення результатів їх виконання,
+      зазвичай використовується у [командній оболонці](#shell).
```

Comment:
TODO: New sentence is better

Reply:
@sofiiamatios @VeronikaShevc @ChubOlya - made a separate sentence. Please approve.